press/utils/__init__.py

```diff
@@ -137,6 +137,13 @@ def _system_user():
 	)
 
 
+def has_role(role, user=None):
+	if not user:
+		user = frappe.session.user
+
+	return frappe.db.exists("Has Role", {"parent": user, "role": role})
```

Comment:
```suggestion
	return frappe.db.exists("Has Role", {"parenttype": "User", "parent": user, "role": role})
```

Has Role is a child table for multiple doctypes. (`/app/doctype?fieldtype=Table&options=Has+Role`)